convert_to_multi_instance.sh

```diff
@@ -53,6 +60,205 @@ function prepare_cluster() {
     # uninstall singleton services
 }
 
+function collect_data() {
+    title "collecting data"
+    msg "-----------------------------------------------------------------------"
+
+    cs_operator_namespace=$(${OC} get pod --all-namespaces | grep operand-deployment-lifecycle-manager | awk '{print $1}')
```

Comment:
change variable name to `master_ns` because `cs_operator_namespace` is a bit misleading